src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -486,4 +496,16 @@ protected function getTemplate(
 
         return $template;
     }
+
+    public function deprecateTemplate($oldTemplateName, $newTemplateName)
+    {
+        @trigger_error(sprintf(
+            'The "%s" template is deprecated since version 3.x and will be removed in 4.0.'
```

Comment:
I don't think we can put version numbers here, unless maybe if we pass one of them as argument. All Sonata bundles are not 3.x.

Reply:
so I can just write `The "%s" template is deprecated` or we should add more parameters?

Reply:
I think you can compute the next major version number with just one parameter.